registry.k8s.io/images/k8s-staging-cluster-api/images.yaml

```diff
@@ -467,6 +469,7 @@
     "sha256:b5ddc8ad1dbcba707761ddac484e582737d0ae236f359b92fc93e63502e4758b": ["v1.11.0-rc.0"]
     "sha256:486162f59f1ce8fb54f15ceac5489441da2e5bab43804cfd6ac6707de55bfe67": ["v1.11.0-rc.1"]
     "sha256:c017edefd6eea25e0726a5fe3c35df3ca644814af69ddc9664119f8ca47d8dcc": ["v1.11.1"]
+    "sha256:d8f52e5af431ef31d10ddda3ccb950ec4d0a57eb0d94831d6fe5cf9054f25c27": ["v1.11.2"]
```

Comment:
SHA `d8f52e5af431` matches kubeadm-bootstrap-controller v1.11.2 in the [staging repo](https://console.cloud.google.com/gcr/images/k8s-staging-cluster-api/global/kubeadm-bootstrap-controller@sha256:d8f52e5af431ef31d10ddda3ccb950ec4d0a57eb0d94831d6fe5cf9054f25c27/details).